### PR TITLE
Feature/ES 16 86a43gm56 Create discount method

### PR DESCRIPTION
**Pull request by mtatiana115**
Create discount method

### CODE REVIEW

File: src/main/java/com/riwi/localstorage/riwi_local_storage/api/controllers/DiscountController.java [hunk 1]
```diff
@@ -20,9 +21,12 @@
 
 import com.riwi.localstorage.riwi_local_storage.api.dto.errors.ErrorResponse;
 import com.riwi.localstorage.riwi_local_storage.api.dto.errors.ErrorsResponse;
+import com.riwi.localstorage.riwi_local_storage.api.dto.request.create.DiscountRequest;
+import com.riwi.localstorage.riwi_local_storage.api.dto.request.create.MembershipRequest;
```
Comment (SantiagoProgramador) [inline src/main/java/com/riwi/localstorage/riwi_local_storage/api/controllers/DiscountController.java]: Unused request of a entity who no belong in the discount controller

File: src/main/java/com/riwi/localstorage/riwi_local_storage/api/controllers/DiscountController.java [hunk 2]
```diff
@@ -20,9 +21,12 @@
...
 import com.riwi.localstorage.riwi_local_storage.api.dto.request.update.DiscountRequestUpdate;
 import com.riwi.localstorage.riwi_local_storage.api.dto.request.update.EntityEnabledRequest;
 import com.riwi.localstorage.riwi_local_storage.api.dto.response.DiscountResponse;
+import com.riwi.localstorage.riwi_local_storage.api.dto.response.MembershipResponse;
```
Comment (SantiagoProgramador) [inline src/main/java/com/riwi/localstorage/riwi_local_storage/api/controllers/DiscountController.java]: delete membership dtos in imports

File: src/main/java/com/riwi/localstorage/riwi_local_storage/infrastructure/services/DiscountService.java [hunk 3]
```diff
@@ -31,8 +31,9 @@ public class DiscountService implements IDiscountService {
 
   @Override
   public DiscountResponse create(DiscountRequest request) {
```
Comment (SantiagoProgramador) [inline src/main/java/com/riwi/localstorage/riwi_local_storage/infrastructure/services/DiscountService.java:33]: create method is missing the code validation, discount entity should not have repeated code. 
Use validateDiscountRequest method below.

**Review by SantiagoProgramador — CHANGES_REQUESTED** [review]
Commit the requested changes to merge the branch

**Comment by mtatiana115** [comment]
done

File: src/main/java/com/riwi/localstorage/riwi_local_storage/api/controllers/DiscountController.java (hunk 2 above)
Comment (mtatiana115) [reply src/main/java/com/riwi/localstorage/riwi_local_storage/api/controllers/DiscountController.java]: done

**Comment by SantiagoProgramador** [comment]
good job

**Pull request merged**